File: calculator.py
print("Enter 2 numbers below")
a = int(input("Enter Number 1 :"))
b = int(input("Enter Number 2 :"))
ch = 0
while ch < 5 :
    print('Calculator Menu')
    print('1.ADD')
    print('2.SUBTRACT')
    print('3.MULTIPLY')
    print('4.DIVIDE')
    print('5.EXIT')

    ch = int(input("Enter Choice (1-5):"))
    break


if ch == 1 :
    c = a + b
    print("Sum :", c)
elif ch == 2 :
    c = a - b
    print("Difference :", c)
elif ch == 3 :
    c = a * b
    print("Product :", c)
elif ch == 4 :
    c = a/b
    print('Quotient :', c)
elif ch == 5:
    print('Exited')

    

File: calculator.rb
puts "Ruby, welcomes you to it's calculator :)"
puts "Enter 2 numbers below :"
a = gets.chomp.to_f
b = gets.chomp.to_f
ch = 0

puts "Calculator Menu"
puts "1. ADD"
puts "2. SUBTRACT"
puts "3. MULTIPLY"
puts "4. DIVIDE"
puts "5. EXIT"

puts "Enter a choice from 1 to 5 as shown above :"

ch = gets.chomp.to_i
    

if ch == 1 
    c = a + b
    puts "The Sum is :#{c}"
elsif ch == 2 
    c = a - b
    puts "The Difference is :#{c}"
elsif ch == 3  
    c = a * b 
    puts "The Product is :#{c}"
elsif ch == 4  
    c = a / b
    puts "The Division is :#{c}"
elsif ch == 5 
    puts "Exited"
else
    puts "Invalid Operation"
end


puts "Visit Again !!!"

